index.html

```diff
@@ -3,27 +3,101 @@
+                    <div class="player-card-stats__key">DEFENCE</div>
+                    <div class="player-card-stats__value">98</div>
+                </div>
+                <div class="player-card-stats player-card-stats__passes">
+                    <div class="player-card-stats__key">PASSES</div>
+                    <div class="player-card-stats__value">95</div>
+                </div>
+                <div class="player-card-stats player-card-stats__shots">
+                    <div class="player-card-stats__key">SHOTS</div>
+                    <div class="player-card-stats__value">99</div>
+                </div>
+            </div>
+            <div class="player-card player-card__best-player-card">
+                <div class="player_card__photo">
+                    <img src="assets/player-photos/AG.png" style="float: top">
+                </div>
+                <div class="player-card__name">K. Bęgowski</div>
+                <div class="player-card-stats player-card-stats__defence">
+                    <div class="player-card-stats__key">DEFENCE</div>
+                    <div class="player-card-stats__value">20</div>
+                </div>
+                <div class="player-card-stats player-card-stats__passes">
+                    <div class="player-card-stats__key">PASSES</div>
+                    <div class="player-card-stats__value">100</div>
+                </div>
+                <div class="player-card-stats player-card-stats__shots">
+                    <div class="player-card-stats__key">SHOTS</div>
+                    <div class="player-card-stats__value">100</div>
+                </div>
+            </div>
+            <div class="player-card player-card__best-player-card">
+                <div class="player_card__photo">
+                    <img src="assets/player-photos/MS.png" style="float: top">
+                </div>
+                <div class="player-card__name">M. Saddam</div>
+                <div class="player-card-stats player-card-stats__defence">
+                    <div class="player-card-stats__key">DEFENCE</div>
+                    <div class="player-card-stats__value">90</div>
+                </div>
+                <div class="player-card-stats player-card-stats__passes">
+                    <div class="player-card-stats__key">PASSES</div>
+                    <div class="player-card-stats__value">50</div>
+                </div>
+                <div class="player-card-stats player-card-stats__shots">
+                    <div class="player-card-stats__key">SHOTS</div>
+                    <div class="player-card-stats__value">101</div>
+                </div>
+            </div>
+        </div>
     </section>
-    <footer>
-        <p>Copyright reserved</p>
-        <p>This project is created and owned by Krzysztof Zając</p>
-        <p>Mentored by Mateuszek</p>
+    <footer class="footer">
+        <p>Copyright reserved &copy;<br />
```

Comment:
I don't like solution breaking code by <br>, you give some responsibility for styling to HTML elements, it. should be done in CSS, you can wrap those elements and according to these elements styled